.gitignore

```diff
@@ -23,6 +23,9 @@ pids
 # Directory for instrumented libs generated by jscoverage/JSCover
 lib-cov
 
+## Reports ##
+report
+
```

Comment:
You already have it, no need to add it again

Reply:
I'll fix it

Reply:
Fixed, can you check again please?